lab_1/start.py

```diff
@@ -29,4 +29,4 @@
     
     RESULT = concordance[:3]
     # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
-    assert RESULT == [['presented', 'wereare'], ['persuaded', 'death'], ['among', 'had']], 'Concordance not working'
+    assert RESULT, 'Concordance not working'

```

Comment:
assert RESULT == [expected result]

I think you've put adjacent words here before, not concordance. Look what you print and just paste this result